packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -576,23 +654,37 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
     }
 
     private getTimeForX(x: number): bigint {
-        if (!this.chartRef.current?.chartInstance) {
-            return BigInt(0);
+        if (this.isBarPlot) {
+            if (!this.chartRef.current) {
+                return BigInt(0);
+            }
+        } else {
+            if (!this.chartRef.current?.chartInstance) {
+                return BigInt(0);
+            }
```

Comment:
Since we're not using the chart anymore, maybe we can skip these checks altogether?

Reply:
Done.